modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -4034,6 +4034,27 @@ public IgniteInternalFuture<?> resetCacheState(Collection<String> cacheNames) th
         return initiateCacheChanges(reqs).stream().collect(IgniteCollectors.toCompoundFuture());
     }
 
+    /**
+     * Attempts to register a cache locally if it has not already been registered.
+     *
+     * @param cacheNames Cache names.
+     * @return Future that will be completed when all caches are checked.
+     */
+    public IgniteInternalFuture<?> checkCaches(Collection<String> cacheNames) {
+        List<IgniteInternalFuture<?>> futures = new ArrayList<>();
+
+        for (String cacheName : cacheNames) {
+            IgniteInternalCache<Object, Object> cache0 = internalCache(cacheName);
+
+            // Try to start cache, there is no guarantee that cache will be instantiated.
+            if (cache0 == null) {
```

Comment:
Please do not use brackets for a single line `if` statement.

Reply:
Fixed